src/pages/signin/SignInPageLayout/SignInPageLayoutWide.js

```diff
@@ -1,92 +1,104 @@
 import React from 'react';
-import {
-    Image, Text, View,
-} from 'react-native';
+import {Image, View} from 'react-native';
 import PropTypes from 'prop-types';
 import styles from '../../../styles/styles';
 import ExpensifyCashLogo from '../../../components/ExpensifyCashLogo';
-import welcomeScreenshot from '../../../../assets/images/welcome-screenshot-wide.png';
+import Text from '../../../components/Text';
+import welcomeScreenshot from '../../../../assets/images/welcome-screenshot.png';
 import variables from '../../../styles/variables';
 import TermsAndLicenses from '../TermsAndLicenses';
-import WelcomeText from '../../../components/WelcomeText';
-import TextLink from '../../../components/TextLink';
 import CONST from '../../../CONST';
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import TextLink from '../../../components/TextLink';
 
 const propTypes = {
     /** The children to show inside the layout */
     children: PropTypes.node.isRequired,
 
-    /** Whether we should show the welcome text (the welcome screenshot always displays on wide views) */
-    shouldShowWelcomeText: PropTypes.bool,
-
     ...withLocalizePropTypes,
 };
 
-const defaultProps = {
-    shouldShowWelcomeText: true,
-};
-
 const SignInPageLayoutWide = props => (
     <View style={[styles.signInPageInner]}>
-        <View style={[styles.flex1, styles.flexRow]}>
-            <View style={[styles.flex1, styles.w50, styles.alignItemsCenter]}>
-                <View>
-                    <Image
-                        resizeMode="contain"
-                        style={[styles.signinWelcomeScreenshotWide]}
-                        source={welcomeScreenshot}
-                    />
+        <View style={[styles.flex1, styles.flexRow, styles.dFlex, styles.flexGrow1]}>
+            <View style={[styles.signInPageWideLeftContainer, styles.dFlex, styles.flexColumn, styles.ph6]}>
+                <View style={[
+                    styles.flex1,
+                    styles.dFlex,
+                    styles.flexColumn,
+                    styles.mt40Percentage,
```

Comment:
Instead of using 40% as the margin top, what if we did something that was actually responsive to the viewport - so something like `15vh`? Does React Native work well with `vh` units?

Reply:
No react-native doesn't work with `vh`. Though a workaround is possible but might require more effort.
`%` somewhat works in a similar fashion

Reply:
Hmm it didn't seem like the margin was actually being responsive though, it felt fixed when resizing the page. Any ideas?

Reply:
To solve this we need to apply different styles based on the platform. Won't be maintainable in the long run.

Reply:
Can you elaborate on what you mean? The left area of the sign up page is basically the same view used on desktop or mobile. The only difference is that on desktop, there is a blue area added to the right of it.